firmware/ec/OpenCellular.cfg

```diff
@@ -574,10 +574,10 @@ m3Hwi1Params.instance.name = "m3Hwi1";
 Program.global.m3Hwi1 = m3Hwi.create(60, "&uDMAIntHandler", m3Hwi1Params);
 */
 
-var m3Hwi2Params = new m3Hwi.Params();
+/*var m3Hwi2Params = new m3Hwi.Params();
```

Comment:
Why are these commented out? Can they either be deleted or can you add a comment describing why they were removed?

Reply:
this configuration has some conflict with SPI DMA, doesn't work with it...

Reply:
Can you add that as a comment in the file?

Reply:
Thanks for adding the comments and test plan